packages/web-api-scan-request-sender/src/sender/on-demand-dispatcher.ts

```diff
@@ -40,6 +42,7 @@ export class OnDemandDispatcher {
                 itemCount = response.item.length;
                 if (itemCount > 0) {
                     await this.sender.sendRequestToScan(response.item);
+                    queuedItemCount += itemCount;
                     this.logger.logInfo(`[Sender] Queued ${itemCount} scan requests to the queue`);
```

Comment:
can we move event here to have detailed monitoring

Reply:
good point. done.